src/admin/users/views/UserOverview.tsx

```diff
@@ -696,42 +693,28 @@ const UserOverview: FunctionComponent<UserOverviewProps & UserProps> = ({ user }
 					title={t('admin/users/views/user-overview___verwijder-opties')}
 					isOpen={deleteOptionsModalOpen}
 					onClose={() => setDeleteOptionsModalOpen(false)}
-					size="large"
+					size="medium"
 				>
 					<ModalBody>
-						<Grid className="a-delete-user__grid">
-							<Column size="3-6">
-								<div
-									className={classnames(
-										'a-delete-user__image',
-										selectedDeleteOption
-									)}
-								/>
-							</Column>
-							<Column size="3-6">
-								<RadioButtonGroup
-									options={GET_DELETE_RADIO_OPTIONS()}
-									value={selectedDeleteOption}
-									onChange={setSelectedDeleteOption as any}
-								/>
-								{(selectedDeleteOption === 'TRANSFER_PUBLIC' ||
-									selectedDeleteOption === 'TRANSFER_ALL') && (
-									<ContentPicker
-										allowedTypes={['PROFILE']}
-										onSelect={(option) =>
-											setTransferToUser(option || undefined)
-										}
-										initialValue={transferToUser}
-										placeholder={t(
-											'admin/users/views/user-overview___overdragen-naar-gebruiker'
-										)}
-										hideTargetSwitch
-										hideTypeDropdown
-										errors={transferToUserError ? [transferToUserError] : []}
-									/>
+						<RadioButtonGroup
+							options={GET_DELETE_RADIO_OPTIONS()}
+							value={selectedDeleteOption}
+							onChange={setSelectedDeleteOption as any}
+						/>
+						{(selectedDeleteOption === 'TRANSFER_PUBLIC' ||
+							selectedDeleteOption === 'TRANSFER_ALL') && (
+							<ContentPicker
+								allowedTypes={['PROFILE']}
+								onSelect={(option) => setTransferToUser(option || undefined)}
```

Comment:
Persoonlijk zou ik dit aanpassen naar `onSelect={setTransferToUser}` en de undefined afhandelen in de functie om zoveel mogelijk de logica binnen een template te vermijden als best practice.